webknossos/webknossos/skeleton/tree.py

```diff
@@ -114,7 +114,7 @@ def __init__(
         group: "Group",
         skeleton: "Skeleton",
         color: Optional[Vector4] = None,
-        enforced_id: Optional[int] = None,  # pylint: disable=unused-argument
```

Comment:
is https://docs.astral.sh/ruff/rules/unused-function-argument/ not enabled? I think, this rule is important because it highlights possible smells. in this case, it's probably due to inheritance..

Reply:
Done